api_specs/v2/component_analyses.yaml

```diff
@@ -52,50 +136,19 @@ paths:
           content:
             application/json:
               schema:
+                type: array
```

Comment:
Why is this one of type array

Reply:
Oh..that change was accidental. No significant changes/proposals in Singular CA call in this iteration.